examples/simple/README.md

```diff
@@ -0,0 +1,2 @@
+change current directory to this directory
+run "node ../../bin/kolint.js view.html"
```

Comment:
Line breaks in markdown are double newlines in raw text, so I changed it to a list, and added the title with a description.

```suggestion
# Simple
Simple example without any errors.

- Change current working directory to `./examples/simple`.
- Run `node ../../bin/kolint.js view.html`
```